web/ui/react-app/src/themes/_shared.scss

```
@@ -104,6 +104,15 @@ button.execute-btn {
   margin-bottom: 20px;
 }
 
+.navbar-brand img {
+  padding-right: 1rem;
+  height: 1.9rem;
+}
+
+.navbar-brand {
+  margin-top: 2px;
```

Currently the bottom line of the "Prometheus" text is a bit above the rest of the nav item texts, which looks a bit buggy. The best possible compromise that I've found while playing around so far is:

```suggestion
  margin-top: 4px;
```

---

done